docs/source/workshops/bioc-2015.rst

```diff
@@ -45,114 +32,87 @@ Install gcloud
 Configure gcloud compute
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. container:: toggle
-
-    .. container:: header
-
-        There are a few steps to do this.  Click here to **Show/Hide Instructions**
-
-    .. container:: content
-
-      .. include:: /includes/gcloud_compute_configure.rst
+.. include:: /includes/gcloud_compute_configure.rst
 
 Spin up a Google Compute Engine Instance running Bioconductor
 -------------------------------------------------------------
 
-This codelab can be run from anywhere.  All the heavy lifting is happening in the cloud regardless of where R is running.  But if you like, you can run the whole darn thing in the cloud.
+This codelab `can be run from anywhere <https://github.com/googlegenomics/bioconductor-workshop-r>`_.  All the heavy lifting is happening in the cloud regardless of where R is running.  But if you like, you can run the whole darn thing in the cloud.
```

Comment:
Besides what Matt mentioned, it probably would be nice to refer to it as a fail-safe environment that is "enabled as a turn-key solution for allowing continous analysis to be performed as well the option of check-pointing the work enviroment for returning back to it in the same state of analysis that it was left in."  This is a powerful concept that many bioinformaticians will appreciate.